src/test/ui/or-patterns/exhaustiveness-unreachable-pattern.stderr

```diff
@@ -77,15 +77,15 @@ LL |         (1 | 1,) => {}
    |              ^
 
 error: unreachable pattern
-  --> $DIR/exhaustiveness-unreachable-pattern.rs:53:15
+  --> $DIR/exhaustiveness-unreachable-pattern.rs:55:15
    |
-LL |             | 0
+LL |             | 0] => {}
```

Comment:
I'm not sure how much we care about emitting match errors in the order in which they occur, but if it's easy to make prior patterns warn earlier, that's probably slightly better for the user.

Reply:
It's quite easy yes, it appears `Span`s are `Ord` wrt their position in a file.